jangaroo-maven/jangaroo-maven-plugin/src/main/java/net/jangaroo/jooc/mvnplugin/AbstractCompilerMojo.java

```diff
@@ -385,8 +396,9 @@ protected JoocConfiguration createJoocConfiguration(Log log) throws MojoExecutio
     configuration.setOutputDirectory(getClassesOutputDirectory());
     configuration.setLocalizedOutputDirectory(getLocalizedOutputDirectory());
     configuration.setApiOutputDirectory(getApiOutputDirectory());
-    configuration.setSourcesAreTests(isTestRun());
-    configuration.setFindUnusedDependencies(findUnusedDependencies());
+    configuration.setFindUnusedDependencies(findUnusedDependencies(staleMillis));
+    //todo: use proper directory
+    configuration.setWarningsOutputDirectory("dependencyWarnings");
```

Comment:
I thought the options were
- either a warnings output directory with multiple text files with well-known names
- or a single JSON file for the report
Now you have a directory with one file with a well-known name.
The configuration should be the File, not the directory.

Reply:
Do you have a recommendation for a default path?

Reply:
Not really, you could put it in a `target/dependencyReports` sub-directory in order not to "pollute" the top-level `target` directory, but since it is just one file, it does not really make a difference.

Reply:
Done